src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -117,11 +117,6 @@ public String getSubmitAction() {
         return submitAction;
     }
     
-    public String getSubmitActionQuestion() {
```

Comment:
Why is this removed? This should be used in `feedbackQuestionSubmissionForm.tag`?